playbooks/roles/backups/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+BACKUPS_MYSQL: yes
+BACKUPS_MONGO: yes
+BACKUPS_BUCKET: ''
+BACKUPS_AWS_ACCESS_KEY_ID: ''
+BACKUPS_AWS_SECRET_ACCESS_KEY: ''
+BACKUPS_DIR: '/tmp/db_backups'
+
+# gs: Google Cloud Storage
+# s3: Amazon Simple Storage Service
+BACKUPS_PROVIDER: 'gs'
+
+
+# Google Cloud Storage
+BACKUPS_GS_BOTO_FILE: '/root/.boto'
+BACKUPS_GS_DEFAULT_PROJECT_ID: ''
+BACKUPS_GS_SERVICE_KEY_CONTENTS: ''
+BACKUPS_GS_SERVICE_KEY_FILE: '/root/backup-service-account.p12'
```

Comment:
/etc/gcloud seems okay. I'm not sure if they already have a standard place where they look for it? (for instance when using the cli?)